### DISCUSSION TITLE
Using redis as a store messages and sessions

### DISCUSSION THREAD

**Discussion by busy-spin** [quickfix-j/quickfixj#415] [Ideas]
Hi 

I would like to get community opinion about using redis as a store for messages and sessions.

I haven't really benchmark, but is there anyone who has tried this ?. 

Would you see this as an interesting POC, I would like to give it a shot and PR.

**Comment by chrjohn** [comment]
IIRC I've heard of other people using this. So it probably would make sense to integrate it into QFJ.
But I'd also like to hear other people's opinion on this.
